packages/openapi2-parser/lib/parser.js

```diff
@@ -103,6 +103,15 @@ class Parser {
       return done(null, this.result);
     }
 
+    if (loaded.swagger === undefined) {
+      this.createAnnotation(
+        annotations.VALIDATION_ERROR, null,
+        "Missing required key 'swagger'"
```

Comment:
nit: should it be quoted with " or ' ? we use ' elsewhere.

also, above there is a string wrapped with parentheses... like `('xyz')`.

Reply:
Agreed, using this style in openapi3-parser. Didn't realise the inconsistency at the time.